# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/tests/css/css-transforms/transform-box/svgbox-stroke-box-005.html
# Commit: ec44194caefb
# Full Hash: ec44194caefb41931247d18ad886326cf6d89a93
# Author: Boris Chiou <boris.chiou@gmail.com>
# Date: 2024-03-05 09:48:50
# Description:
#   Bug 1870200 - Break the cyclic dependency if any of the desendants uses non-scaling-stroke. r=emilio
#   
#   While we are computing the transform-box:stroke-box, for CSS Transforms or
#   Motion path Transforms, we have to avoid the cyclic dependency not only
#   for the SVG geometry frame itself, but also for the desendants of
# ==============================================================================

diff -r 5c84d062c789 -r ec44194caefb testing/web-platform/tests/css/css-transforms/transform-box/svgbox-stroke-box-005.html
--- /dev/null	Thu Jan 01 00:00:00 1970 +0000
+++ b/testing/web-platform/tests/css/css-transforms/transform-box/svgbox-stroke-box-005.html	Mon Mar 04 20:46:05 2024 +0000
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<title>transform-box: border-box, stroke with vector-effect: non-scaling-stroke</title>
+<link rel="match" href="reference/svgbox-rect-ref.html">
+<link rel="help" href="https://drafts.csswg.org/css-transforms-1/#transform-box">
+<link rel="help" href="https://svgwg.org/svg2-draft/coords.html#VectorEffects">
+<link rel="help" href="https://github.com/w3c/csswg-drafts/issues/9640">
+<meta name="assert" content="The used value of transform-box is fill-box on SVG with non-scaling-stroke"/>
+<style>
+#container {
+  fill: green;
+  stroke: black;
+  stroke-width: 20;
+  transform-box: border-box;
+  transform: scale(0.5) translateY(-100%);
+}
+#inner {
+  vector-effect: non-scaling-stroke;
+}
+</style>
+<svg width="400" height="300">
+  <a id="container">
+    <rect id="inner" width="100" height="200" x="50" y="180"/>
+  </a>
+</svg>
